saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
 msgid "Manager"
 msgstr "Directeur/Directrice"
 
-#: internal_ops/forms.py:47 internal_ops/forms.py:180
-msgid "Date Manager Approved the request"
-msgstr "Date à laquelle le responsable a approuvé la demande"
+#: internal_ops/forms.py:42 internal_ops/forms.py:172 internal_ops/forms.py:273
+#: internal_ops/forms.py:375
+msgid "Date Manager reviewed the request"
+msgstr "Date à laquelle le responsable a examiné la demande"
 
-#: internal_ops/forms.py:48 internal_ops/forms.py:181
+#: internal_ops/forms.py:43 internal_ops/forms.py:167 internal_ops/forms.py:274
+#: internal_ops/forms.py:376
+msgid "Manager Approved"
+msgstr "Gestionnaire approuvé"
+
+#: internal_ops/forms.py:44 internal_ops/forms.py:168 internal_ops/forms.py:275
+#: internal_ops/forms.py:377
+msgid "Manager Denied"
+msgstr "Gestionnaire refusé"
+
+#: internal_ops/forms.py:45 internal_ops/forms.py:173
 msgid "Submitted By"
 msgstr "Proposé par"
 
-#: internal_ops/forms.py:49 internal_ops/forms.py:182
+#: internal_ops/forms.py:46 internal_ops/forms.py:169 internal_ops/forms.py:174
+#: internal_ops/forms.py:276 internal_ops/forms.py:378
 msgid "Date Info Requested"
 msgstr "Informations sur la date demandées"
 
-#: internal_ops/forms.py:50 internal_ops/forms.py:183
+#: internal_ops/forms.py:47 internal_ops/forms.py:170 internal_ops/forms.py:175
+#: internal_ops/forms.py:277 internal_ops/forms.py:379
 msgid "Info Requested"
 msgstr "Informations demandées"
 
-#: internal_ops/forms.py:56 internal_ops/forms.py:184
+#: internal_ops/forms.py:48 internal_ops/forms.py:176 internal_ops/forms.py:279
+#: internal_ops/forms.py:381
 msgid "Fund Center"
 msgstr "Centre de fonds"
```

Comment:
I'd have to check with official documentation as I'm not sure it's the same term used in French